env.example

```diff
@@ -11,9 +11,17 @@ MESSAGE_STATUS_FOR_VIEW_TOPIC_CONSUMER_CONNECTION_STRING=<>
 COSMOSDB_PORT=<PORT>
 COSMOSDB_KEY=<KEY>
 COSMOSDB_NAME=<NAME>
-COSMOSDB_URI=<UTI>
+COSMOSDB_URI=<URI>
 COSMOSDB_CONNECTION_STRING=<CONNECTION_STRING>
 
+REMOTE_CONTENT_COSMOSDB_URI=<URI>
+REMOTE_CONTENT_COSMOSDB_KEY=<KEY>
+REMOTE_CONTENT_COSMOSDB_NAME=<NAME>
+REMOTE_CONTENT_COSMOSDB_CONNECTION_STRING=<CONNECTION_STRING>
+
+MESSAGE_CONFIGURATION_CHANGE_FEED_LEASE_PREFIX=cf-prefix
```

Comment:
Do we need to add `MESSAGE_CONFIGURATION_CHANGE_FEED_LEASE_PREFIX` in `config.ts`?